src/views/Order/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['React' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)